examples/variational_autoencoders/vae.py

```diff
@@ -17,25 +17,28 @@
 
 
 @zs.reuse('model')
-def vae(observed, n, x_dim, z_dim, n_particles):
+def vae(observed, x_dim, z_dim, n_z_per_x):
     with zs.BayesianNet(observed=observed) as model:
-        z_mean = tf.zeros([n, z_dim])
-        z = zs.Normal('z', z_mean, std=1., group_ndims=1, n_samples=n_particles)
+        z_mean = tf.zeros([1, z_dim])
+        z = zs.Normal('z', z_mean, std=1., group_ndims=1, n_samples=n_z_per_x)
         lx_z = tf.layers.dense(z, 500, activation=tf.nn.relu)
         lx_z = tf.layers.dense(lx_z, 500, activation=tf.nn.relu)
         x_logits = tf.layers.dense(lx_z, x_dim)
-        x = zs.Bernoulli('x', x_logits, group_ndims=1)
-    return model, x_logits
+        x_mean = zs.Implicit("x_mean", x_logits, group_ndims=1)
```

Comment:
If called `x_mean`, then `tf.sigmoid(x_logits)` is better.

Reply:
Yes, I agree I'll change that as well.